### PR TITLE
apk: add Spanish translation

### PR DESCRIPTION
**Pull request by ap-ch**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/linux/apk.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# apk
+
+> Herramienta de gestión de paquetes de Alpine Linux.
+> Más información: <https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management>.
+
+- Actualizar los índices de repositorio desde todos los repositorios remotos:
+
+`apk update`
+
+- Instalar un nuevo paquete:
```
Comment (navarroaxel) [inline pages.es/linux/apk.md:10]: For Romance languages, the present indicative fits better.
```suggestion
- Instala un nuevo paquete:
```

File: pages.es/linux/apk.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`apk add {{paquete}}`
+
+- Eliminar un paquete:
```
Comment (navarroaxel) [inline pages.es/linux/apk.md:14]:
```suggestion
- Remueve un paquete:
```

File: pages.es/linux/apk.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`apk del {{paquete}}`
+
+- Reparar paquete o actualizarlo sin modificar dependencias principales:
```
Comment (navarroaxel) [inline pages.es/linux/apk.md:18]: This is an old page, we improve our grammar rules in the time. The English page needs to be fixed too
```suggestion
- Repara un paquete o lo actualiza sin modificar dependencias principales:
```

File: pages.es/linux/apk.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`apk fix {{paquete}}`
+
+- Buscar paquete mediante palabra clave:
```
Comment (navarroaxel) [inline pages.es/linux/apk.md:22]:
```suggestion
- Busca un paquete usando palabras clave:
```

File: pages.es/linux/apk.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`apk search {{palabra clave}}`
```
Comment (navarroaxel) [inline pages.es/linux/apk.md:24]: We use the placeholders in `snake_case`
```suggestion
`apk search {{palabra_clave}}`
```

File: pages.es/linux/apk.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Obtener información sobre un paquete en específico:
```
Comment (navarroaxel) [inline pages.es/linux/apk.md:26]:
```suggestion
- Muestra información sobre un paquete específico:
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

> Welcome to tldr btw

**Pull request merged**